src/unix/glegl.cpp

```diff
@@ -364,9 +364,51 @@ bool wxGLCanvasEGL::InitVisual(const wxGLAttributes& dispAttrs)
     return m_config != nullptr;
 }
 
+
+extern "C"
+{
+
+typedef EGLDisplay (*GetPlatformDisplayFunc)(EGLenum platform,
+                                             void* native_display,
+                                             const EGLAttrib* attrib_list);
+
+// Returns the appropriate function pointer for "eglGetPlatformDisplay()" or
+// "eglGetPlatformDisplayEXT()" or nullptr if neither is available at runtime.
+static GetPlatformDisplayFunc get_edl_platform_display_func()
```

Comment:
Presumably you meant to name this `get_egl_platform_display_func` ?

Reply:
I ended up removing this in favor of embedding the static variable in `GetDisplay()` since I now need to call `wxGLCanvasBase::IsExtensionInList()`, which is protected. I can't use `wxGLCanvasBase::IsExtensionSupported()` from `wxGLCanvasEGL::GetDisplay()` because the former calls the latter.